.github/workflows/postman-docker.yaml

```diff
@@ -22,38 +22,56 @@ jobs:
 
       matrix:
         backend: [ cassandra-40, cassandra-311, dse-68 ]
-        collection: [ docsapi-library, graphqlapi-library, restapi-users ]
+        collection: [ graphqlapi-library, restapi-users ]
         include:
           - backend: cassandra-40
             path: docker-compose/cassandra-4.0
           - backend: cassandra-311
             path: docker-compose/cassandra-3.11
           - backend: dse-68
             path: docker-compose/dse-6.8
-          - collection: docsapi-library
-            collection-id: 12949543-caba2a02-6559-486a-9e4a-d5c0791fd296
-            environment-id: 17930693-2f355f7b-123d-4bca-9a9c-7d82222ca49d
+          #- collection: docsapi-library
+          #  collection-id: 12949543-caba2a02-6559-486a-9e4a-d5c0791fd296
+          #  environment-id: 12949543-2e78cf27-bd8c-43f2-909f-70a2b87d65fe
           - collection: graphqlapi-library
             collection-id: 17930693-65da5c64-561a-449b-a0e8-0318575f6871
-            environment-id: 17930693-2f355f7b-123d-4bca-9a9c-7d82222ca49d
+            environment-id: 12949543-2e78cf27-bd8c-43f2-909f-70a2b87d65fe
           - collection: restapi-users
             collection-id: 17930693-47ab5f0d-407e-48cf-aa11-c51d129f1eef
-            environment-id: 17930693-2f355f7b-123d-4bca-9a9c-7d82222ca49d
+            environment-id: 12949543-2e78cf27-bd8c-43f2-909f-70a2b87d65fe
+
     steps:
       - uses: actions/checkout@v3
+
       - name: Install Postman CLI
         run: |
           curl -o- "https://dl-cli.pstmn.io/install/linux64.sh" | sh
+
       - name: Login to Postman CLI
         run: postman login --with-api-key ${{ secrets.POSTMAN_API_KEY }}
+
       - name: Start Backend
         # Run Stargate coordinator in developer mode to save time / resources
         run: |
           cd ${{ matrix.path }}
           docker compose -f docker-compose-dev-mode.yml up -d
-      - name: Run API test
-        run: |
-          postman collection run ${{ matrix.collection-id }} -e ${{ matrix.environment-id }}
+
+      #- name: Run API test
+      #  run: |
+      #    postman collection run ${{ matrix.collection-id }} -e ${{ matrix.environment-id }} --verbose --timeout-request 10000
```

Comment:
Currently commented out while I try Newman (OSS CLI for Postman). 
I have tried various settings. Verbose flag allowed me to see the socket close message. Increasing timeout seems not to fix the problem